pymysqlreplication/binlogstream.py

```diff
@@ -35,14 +35,16 @@ class BinLogStreamReader(object):
     """Connect to replication stream and read event
     """
 
-    def __init__(self, connection_settings, server_id, resume_stream=False,
+    def __init__(self, connection_settings, server_id, ctl_connection_settings=None, resume_stream=False,
                  blocking=False, only_events=None, log_file=None, log_pos=None,
                  filter_non_implemented_events=True,
                  ignored_events=None, auto_position=None,
                  only_tables=None, only_schemas=None,
-                 freeze_schema=False, skip_to_timestamp=None):
+                 freeze_schema=False, skip_to_timestamp=None,
+                 fail_on_table_metadata_unavailable=False):
```

Comment:
I think the intention was we eventually wanted to merge this in to the overall master branch of the library, and to do so we want to keep existing functionality exactly the same (the corresponding branch in replhandler sets this to True).

Do we want to make this default to True?